### DISCUSSION TITLE
Welcome to OpenAvalancheProject Discussions!

### DISCUSSION THREAD

**Discussion by scottcha** [scottcha/OpenAvalancheProject#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by snowman907** [comment]
Hello Scott,
I have tried to reach you through different applications. I am working on avalanche data processing project in Alaska, specifically, the Dalton Highway at  Atigun Pass, with the intent to grow the project to other Alaska roads prone to Avalanche.  We are erecting a new local weather station in late September to add better local data with better snow sensors. 

In short, I would like to learn how I can join your project with adding an Alaska component to it.  My background in AI is very limited, I just started taking a class through Coursera (so extremely novice). Is there a way we can get in touch via email (ejsaiet@alaska.edu)?
Thanks
Eyal

**Comment by macleodmore** [comment]
I am trying to use the data you provided in the Readme to run a model and am having trouble running the y_df.p file on my version of matlab. What version did you create it with, apparently there are some backwards compatability issues with that file type.

**Reply by scottcha** [reply]
Hi, nothing in OAP is dependent on matlab.  
I haven't provided a status update in a bit but there is a branch called "add_batch" where more recent changes are stored.  The jist is that with the methodology documented I wasn't able to get any accuracy higher than ~65-70% and have been reengineering the project to try and get around that with better data and less chance for overfitting.  I hope to provide a more comprehensive update in the coming months.

**Comment by macleodmore** [comment]
Interesting, in the .zip file in the Readme document there was a .npy file
and a .p file. I was unfamiliar with both and looked online which said the
.p is a type of Matlab runnable file. If that's not the case, how should I
run or try to open these two files?
